web/sites/default/config/views.view.notifications.yml

```diff
@@ -0,0 +1,236 @@
+uuid: 078c13a6-9bbe-4dd2-ba4b-e5eb66248745
```

Comment:
Please combine both block views into a single view for "Group notifications". Ideally each block will have a meaningful machine name as well instead of block_1. Every additional view config adds up and makes it so there is more separate config to maintain and import, so group whenever possible.